app/main.py

```diff
@@ -1,6 +1,24 @@
+import requests
+from api_key import API_KEY
```

Comment:
Importing the API key directly from `api_key.py` is not secure. According to the task requirements, the API key should be read from an environment variable. Consider using `os.environ.get('API_KEY')` to retrieve the API key securely.